README.md

```diff
@@ -50,6 +50,23 @@
 ### Prerequisites
 
 You can run Application Runner itself, but it is generally built as a library and needs a Phoenix parent application to access all functionalities: 
+
+You will first need to start two databases, postgres and mongo.
+
+command at the root of this project or running by hand the docker commands like so :
```

Comment:
You should add the docker-compose way to start both databases. Do not remove the basic docker way, but let it in second place.